<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookWidget</class>
 <widget class="QWidget" name="BookWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ImageField">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>221</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/41KhAq3f0TL._SX313_BO1,204,203,200_.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="TitleField">
      <property name="text">
       <string>Algortihms and Data Structures</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AuthorField">
      <property name="text">
       <string>Kurth Mehlhorn, Peter Sanders</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PagesField">
      <property name="text">
       <string>312 Pages</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="DateField">
      <property name="text">
       <string>23 June 2008</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
